docker-compose.yml

```diff
@@ -157,7 +155,6 @@ services:
       PGADMIN_DEFAULT_EMAIL: ${EMAIL:-pgadmin4@pgadmin.org}
       PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
       PGADMIN_CONFIG_SERVER_MODE: 'False'
-    restart: unless-stopped
```

Comment:
What's wrong with the restart directive ?

Do you prefer having it in the docker compose override ?

Reply:
It is a duplicated keyvalue (you'll see it below (l~176)